windows/win_regedit.ps1

```diff
@@ -118,8 +118,8 @@ else
 
 }
 
-if($registryDataType -eq "binary" -and $registryData -ne $null) {
-   $registryData = Convert-RegExportHexStringToByteArray($registryData)
+if($registryDataType -eq "binary" -and $registryData -ne $null -and $registryData.GetType().Name -eq 'String') {
```

Comment:
Only reason is I'd not come across the -is [String] syntax before.  It is considerably less ugly!  I will know for next time.